src/OpenLoco/Objects/SteamObject.h

```diff
@@ -6,17 +6,25 @@
 
 namespace OpenLoco
 {
+    namespace SteamObjectFlags
+    {
+        constexpr uint16_t unk0 = (1 << 0);
```

Comment:
any idea what any of these are? unk2 looks like it could be whether the exhaust has sound? unk3 seems to indicate whether it's being drawn at the centre of the image or not?

Reply:
Unk0 is if the exhaust has a horizontal component.

Reply:
sounds good to me, it would be really helpful if the variables could be renamed, even if the names aren't perfect, it adds a lot more info than `unk*`